.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     hooks:
     - id: black
       name: black
-      entry: ./.venv/bin/black
+      entry: black
```

Comment:
was there a reason for this? removing the .venv path makes this work easier with pyenv